src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -1439,28 +1439,61 @@ struct P1_ALIGNAS(4) TroposphereConfig {
  * @ingroup config_and_ctrl_messages
  */
 struct P1_ALIGNAS(4) LBandConfig {
-  /** The center frequency of the L-band beam (Hz). */
-  float center_frequency_hz = NAN;
+  /**
+   * The center frequency of the L-band beam (Hz). If `NAN`, use the system
+   * default value.
+   */
+  double center_frequency_hz = NAN;
 
   /**
    * The size of the signal acquisition search space (in Hz) around the center
-   * frequency.
+   * frequency. If `NAN`, use the system default value.
    *
    * For example, a value of 6000 will search +/- 3 kHz around the center
    * frequency.
    */
   float search_window_hz = NAN;
 
+  /**
+   * If `true`, use the provider date rate in this configuration. Otherwise, use
+   * device's default value.
+   */
+  bool use_custom_pmp_data_rate = false;
+
+  /**
+   * If `true`, use the provider unique word in this configuration. Otherwise,
+   * use device's default value.
+   */
+  bool use_custom_pmp_unique_word = false;
+
+  /**
+   * If `true`, use the descrambler initialization vector in this configuration.
+   * Otherwise, use device's default value.
+   */
+  bool use_custom_descrambler_init = false;
+
+  /**
+   * If `true`, only output data frames with the configured service ID.
+   * Otherwise, output all decoded frames.
+   */
+  bool filter_data_by_service_id = true;
+
+  /** Enable/disable the descrambler. */
+  bool use_descrambler = true;
+
+  uint8_t reserved[1] = {0};
+
   /** Service ID of the provider. */
-  uint32_t pmp_service_id = 0;
+  uint16_t pmp_service_id = 0x5555;
```

Comment:
should we make all of these be the defaults for our service? That seems to make sense to me.